### DISCUSSION TITLE
Get-PSReadLineOption CommandColor is empty

### DISCUSSION THREAD

**Discussion by thetrotfreak** [lzybkr/PSReadLine#3710] [Q&A]
The `CommandColor` property of `Get-PSReadLineOption` is _empty_ whenever it is **Run as administrator.**

1. Here's the output of `Get-PSReadLineOption` when **Run as administrator** (using `conhost.exe`, _Windows Console Host_):
![image](https://github.com/PowerShell/PSReadLine/assets/45330487/b5ed40f0-b695-417f-860c-727a5d1be60b)
- As you can see the commands are not visible.

2. Here's the output of `Get-PSReadLineOption` (using wt.exe, _Windows Terminal_):
![image](https://github.com/PowerShell/PSReadLine/assets/45330487/4e8782cc-1b5a-4cdd-908e-f50421c3c45e)
- Commands are properly visible.

### Clarifications
1. The colors are somehow messed up when the **Terminal Application** used for the **Command Line** is `conhost.exe`. The colors are fine (regardless of whether it's **elevated**) when using through `wt.exe`.

**Comment by daxian-dbw** [comment]
It doesn't reproduce for me. The screenshot is for running elevated conhost.exe with powershell. I'm using the latest PSReadLine version.

![image](https://github.com/PowerShell/PSReadLine/assets/127450/d5e05c09-7765-4351-a4a2-8d0b154022d1)

**Reply by thetrotfreak** [reply]
- I'm quite sure it has something to do when I changed the **Defaults** for `conhost.exe`.

- The colors were fine before I meddled with the **Default Windows Properties**, like `Screen Text` & `Screen Background` under the `Colors` tab for `conhost.exe`.

- See below:
![IMG-20230609-WA0001](https://github.com/PowerShell/PSReadLine/assets/45330487/08ccd8bc-efcc-4b64-896a-619020383347)
- I changed them back to what it was but nothing seems to have an effect. It only got worse now:
![IMG-20230609-WA0002](https://github.com/PowerShell/PSReadLine/assets/45330487/314b6b6b-1260-4ca9-a645-e63bbaa92aee)
- Using `PSReadLine` version `2.2.6`
